sdcm/mgmt.py

```diff
@@ -101,6 +101,22 @@ def from_str(cls, output_str):
             raise ScyllaManagerError("Could not recognize returned task status: {}".format(output_str))
 
 
+class TaskStatusNew:
+    statuses = ("NEW",
+                "RUNNING",
+                "DONE",
+                "UNKNOWN",
+                "ERROR",
+                "STOPPED",
+                "STARTING")
+
+    @classmethod
+    def get_status(cls, status_string):
```

Comment:
I'm not sure why we need this function ? if we move a simple list of strings instead of enum ?
a check can be: 
```status in self.statuses```